src/include/wiredtiger.in

```diff
@@ -799,6 +799,17 @@ struct __wt_session {
 	 */
 	int __F(reconfigure)(WT_SESSION *session, const char *config);
 
+	/*!
+	 * Initiate a single operation to manage shared storage.
+	 *
+	 * @param session the session handle
+	 * @configstart{WT_SESSION.share_storage, see dist/api_data.py}
+	 * @config{force, force sharing of all data., a boolean flag; default \c false.}
+	 * @configend
+	 * @errors
+	 */
+	int __F(share_storage)(WT_SESSION *session, const char *config);
+
```

Comment:
For `share_storage` API, just wondering if we might want finer grain, like a URI argument?

Reply:
I took inspiration from `checkpoint`. It does not have a URI argument. If we want finer grain, I would suggest we add it in a consistent way as `checkpoint` and use `target=` in the configuration string.

Reply:
Works for me.